internal/core/data/event.go

```diff
@@ -131,7 +131,7 @@ func addNew(e models.Event) (string, error) {
 					return "", err
 				}
 			}
-			valid, err := isValidValueDescriptor(vd, e.Readings[reading])
+			valid, _ := isValidValueDescriptor(vd, e.Readings[reading])
```

Comment:
I know it is not directly related to this PR, but can you change the next if to add a test for err != nil

Reply:
Yes, I like your idea.